acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/jsonrpc/Clique.java

```diff
@@ -98,6 +109,10 @@ private int cliqueBlockPeriod(final PantheonNode node) {
     return Arrays.stream(validators).map(PantheonNode::getAddress).sorted().toArray(Address[]::new);
   }
 
+  public Condition proposerAtChainHeadEqual(final PantheonNode proposer) {
```

Comment:
maybe not a clique thing?

Reply:
Think it is, the condition needs to decode the clique specific extra data